test/Makefile

```diff
@@ -1,4 +1,8 @@
-all: r2r-tests unit_tests
+all:
+	export LC_ALL=C
+	$(MAKE) r2r-tests
+	$(MAKE) unit_tests
+	$(MAKE) test-8051-reassembly
```

Comment:
why dont run those 3 targets in parallel as it was doing before? please change this